dockerfiles/Dockerfile.openvino-csharp

```diff
@@ -1,110 +1,91 @@
-    apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB && rm GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB && \
-    echo "deb https://apt.repos.intel.com/openvino/2022 bionic main" | tee /etc/apt/sources.list.d/intel-openvino-2022.list && \
-    apt update -y && \
-    apt -y install openvino-2022.1.0 && \
-    cd ${INTEL_OPENVINO_DIR}/install_dependencies && ./install_openvino_dependencies.sh -y && \
-    cd /opt/libusb-1.0.22/ && \
-    /usr/bin/install -c -m 644 libusb-1.0.pc '/usr/local/lib/pkgconfig' && \
-    cp ${INTEL_OPENVINO_DIR}/runtime/3rdparty/97-myriad-usbboot.rules /etc/udev/rules.d/ && \
-    ldconfig && \
-# Install GPU runtime and drivers
-    cd ${MY_ROOT} && \
-    mkdir /tmp/opencl && \
-    cd /tmp/opencl && \
-    apt update -y && \
-    apt install -y --no-install-recommends ocl-icd-libopencl1 && \
-    rm -rf /var/lib/apt/lists/* && \
-    wget "https://github.com/intel/compute-runtime/releases/download/21.38.21026/intel-gmmlib_21.2.1_amd64.deb" && \
-    wget "https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.8708/intel-igc-core_1.0.8708_amd64.deb" && \
-    wget "https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.8708/intel-igc-opencl_1.0.8708_amd64.deb" && \
-    wget "https://github.com/intel/compute-runtime/releases/download/21.38.21026/intel-opencl_21.38.21026_amd64.deb" && \
-    wget "https://github.com/intel/compute-runtime/releases/download/21.38.21026/intel-ocloc_21.38.21026_amd64.deb" && \
-    wget "https://github.com/intel/compute-runtime/releases/download/21.38.21026/intel-level-zero-gpu_1.2.21026_amd64.deb" && \
-    dpkg -i /tmp/opencl/*.deb && \
-    ldconfig && \
-    rm -rf /tmp/opencl && \
 # Install Mono
-    cd ${MY_ROOT} && \
-    apt install -y gnupg ca-certificates && \
-    curl https://download.mono-project.com/repo/xamarin.gpg | apt-key add - && \
-    echo "deb https://download.mono-project.com/repo/ubuntu stable-bionic main" | sudo tee /etc/apt/sources.list.d/mono-official-stable.list && \
+RUN wget http://download.mono-project.com/repo/xamarin.gpg && apt-key add xamarin.gpg && rm xamarin.gpg && \
+    echo "deb https://download.mono-project.com/repo/ubuntu stable-bionic main" | tee /etc/apt/sources.list.d/mono-official-stable.list && \
     apt update -y && \
-    apt install -y mono-devel && \
+    apt install -y mono-devel
+
 # Install nuget.exe
-    wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe && \
+RUN wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe && \
     mv nuget.exe /usr/local/bin/nuget.exe && \
     echo 'mono /usr/local/bin/nuget.exe $@' > /usr/local/bin/nuget && \
-    chmod a+x /usr/local/bin/nuget && \
+    chmod a+x /usr/local/bin/nuget
+
 # Install .NET core
-    wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb && \
+RUN wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb && \
```

Comment:
Should not we try to put everything on one RUN Command

Reply:
Tried comparing the size of images with and without single RUN command. Since we are using multiple stages during docker build, there is no much difference in the image sizes.